internal/cfgplugins/qos.go

```diff
@@ -260,3 +261,90 @@ func enableQosRemarkDscpCliConfig(dut *ondatra.DUTDevice, classifierName string,
 		return ""
 	}
 }
+
+func CreateQueues(t *testing.T, dut *ondatra.DUTDevice, qos *oc.Qos, queues []string) {
+	for index, q := range queues {
+		queue := qos.GetOrCreateQueue(q)
+		queue.Name = ygot.String(q)
+		if deviations.QOSQueueRequiresID(dut) {
+			queue.QueueId = ygot.Uint8(uint8(index))
+		}
+	}
+}
+
+func CreateQueuesFromCli(t *testing.T, dut *ondatra.DUTDevice, queues []string) {
+	cliConfig := ""
+	for index, queue := range queues {
```

Comment:
Here and throughout, all CLI config needs to be wrapped with a switch dut.Vendor() statement.